lib/elixir/lib/dynamic_supervisor.ex

```diff
@@ -476,46 +551,20 @@ defmodule DynamicSupervisor do
   module-based supervisors. See the "Module-based supervisors" section
   in the module documentation for more information.
 
-  The `options` received by this function are also supported by `start_link/1`.
-
-  This function returns a tuple containing the supervisor options.
+  It accepts the same `options` as `start_link/1` (except for `:name`)
+  and it returns a tuple containing the supervisor options.
 
   ## Examples
 
       def init(_arg) do
         DynamicSupervisor.init(max_children: 1000, strategy: :one_for_one)
```

Comment:
Since `:strategy` is no longer required, does it make sense to simplify the example?

```suggestion
        DynamicSupervisor.init(max_children: 1000)
```